### PR TITLE
Robert allen

### PR DESCRIPTION
**Pull request by robertjallen**

### CODE REVIEW

File: great-idea-website/index.html [hunk 1]
```diff
@@ -15,49 +15,93 @@
...
+    <div class="hero-image">
+      <img src="img/header-img.png" alt="Image of a code snippet.">
+    </div>
+  </div>
+  <hr>
+  
 
-  About
-  Aliquam elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi. Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.
+<div class="section2">
```
Comment (AislynnEdmiston) [inline great-idea-website/index.html:53]: Way to be semantic! Also classes are easy to understand!

File: great-idea-website/index.html [hunk 2]
```diff
@@ -15,49 +15,93 @@
...
+        <nav>
+          <a href="">Services</a>
+          <a href="">Product</a>
+          <a href="">Vision</a>
+          <a href="">Features</a>
+          <a href="">About</a>
+          <a href="">Contact</a>
+        </nav>
+        <img class="logo" src="img/logo.png" alt="Great Idea! Company logo.">
+      </div>
```
Comment (AislynnEdmiston) [inline great-idea-website/index.html:30]: The only thing to complete this is your nav. You are almost there! Keep going!

File: great-idea-website/index.html [hunk 3]
```diff
@@ -15,49 +15,93 @@
...
+    <h2>Services</h2>
+    <p>Aliquam elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi.
+      Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.</p>
+  </section>
+  <section>
+    <h2>Product</h2>
+    <p>Aliquam elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi.
+      Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.</p>
+  </section>
+  <section>
```
Comment (AislynnEdmiston) [inline great-idea-website/index.html:81]: Semantic and easy to read!

File: great-idea-website/index.html (hunk 1 above)
Comment (robertjallen) [reply great-idea-website/index.html:53]: thanx

**Pull request merged**